fetch.bs

```diff
@@ -1410,9 +1413,11 @@ Unless stated otherwise, it is unset.
  <dl>
   <dt>"<code>default</code>"
   <dd><a for=/>Fetch</a> will inspect the HTTP cache on the way to the network.
-  If there is a fresh response it will be used. If there is a stale response a conditional request
-  will be created, and a normal request otherwise. It then updates the HTTP cache with the response.
-  [[!HTTP]] [[!HTTP-SEMANTICS]] [[!HTTP-COND]] [[!HTTP-CACHING]] [[!HTTP-AUTH]]
+  If there is a fresh response it will be used. If there is a <a>stale response inside revalidate window</a>
+  and the request has the <a for=request>stale content mode</a> set it will be returned
```

Comment:
Looks like stale content mode is no longer a boolean but has a value, this text needs to be updated as well?

Reply:
This text is no longer present.